kitsune/users/jinja2/users/includes/auth_contributor.html

```diff
@@ -3,7 +3,6 @@
 <div class="grid_12">
   <article id="register-form" class="main">
     <h1>{{ _('Register to be a Contributor') }}</h1>
-    <span>{{ _('You don\'t need to register to <a href="{url}">ask a question</a>.')|fe(url=url('questions.aaq_step1')) }}</span>
```

Comment:
How does this affect the presence of two elements with ID search-q, as reported in the related issue?